kafka-source/README.adoc

```diff
@@ -0,0 +1,24 @@
+= Spring Cloud Stream Kafka Source
+
+This module receives messages from Kafka topics.
+
+== Requirements
+
+* Java 7 or Above
+
+== Classes
+
+KafkaSourceApplication:: the Spring Boot Main Application
+KafkaSource:: the Kafka source
+
+== Build
+
+```
+$> mvn package
+```
+
+== Run
+
+```
+$> java -jar target/kafka-source-${version}-exec.jar
```

Comment:
It would be good to include the doc on how would someone specify the `partitions` and `initialOffsets` options when running the module here.

For instance with options like these:

```
--topics=test123,test234 --partitions.test123=1,2 --partitions.test234=1
```